### PR TITLE
cwdeng test data

### PR DESCRIPTION
**Pull request by andrewdotn**
This PR:
  - Adds tests for the srseng and arpeng munge code, since shared code is being changed
  - Adds a transliterator for the existing crkeng importjson dictionaries
  - Checks in a test dictionary for cwdeng. No tests actually use this test data  yet, but this way you can quickly get a demo site running with `USE_TEST_DB`.

### CODE REVIEW

File: docs/dictionary-data.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## Current dictionary data
+
+In theory, linguists will provide comprehensive and correct dictionary data
+in importjtson format.
```
Comment (nienna73) [inline docs/dictionary-data.md:4]:
```suggestion
in importjson format.
```

File: docs/dictionary-data.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    checked in to the secret git repository at
+    `altlab.dev:/data/altlab.git`. This is what’s used in production. It
+    was created by importing the old `crkeng.xml` file into an older
+    version of cree-intelligent-dictionary, that did a lot of paradigm and
+    analysis inference during import, and then the database contents were
+    exported in the new importjson format.
+
+  - For Woods Cree, the `cwdize` script transliterates the production
+    `crkeng_dictionary.importjson` file, using the `database.ndjson` file
+    from the ALTLab repo to get the proto-Cre forms.
```
Comment (nienna73) [inline docs/dictionary-data.md:19]:
```suggestion
    from the ALTLab repo to get the proto-Cree forms.
```

File: munge/shared/dictionary.ts [hunk 3]
```diff
@@ -48,50 +50,100 @@ type ExportableWordform<L> = Required<Omit<Wordform<L>, "formOf">> & {
...
+  constructor(lexicalTags?: string[]) {
+    if (lexicalTags) {
+      this._lexicalTags = new Set(lexicalTags);
+    } else {
+      this._lexicalTags = null;
+    }
     this._entries = [];
     this._byText = new Map();
+    this._bySlug = new Map();
+    this._bySlug = new Map();
```
Comment (nienna73) [inline munge/shared/dictionary.ts:87]: Why is this defined twice?

Comment (andrewdotn) [reply munge/shared/dictionary.ts:87]: Good catch! This is an artifact of resolving some slightly hairy merge conflicts across different branches where I was working on this. I will remove it.

```suggestion
```

**Review by nienna73 — APPROVED** [review]

File: src/CreeDictionary/phrase_translate/crk_tag_map.py [hunk 4]
```diff
@@ -13,6 +13,8 @@
     ("+Obv", COPY_TAG_NAME, 1),
     ("+Loc", COPY_TAG_NAME, 1),
     ("+Distr", COPY_TAG_NAME, 1),
+    ## XXX FIXME I have no idea what +Voc is but it causes problems so drop it
```
Comment (dwhieb) [inline src/CreeDictionary/phrase_translate/crk_tag_map.py:16]: `Voc` = [vocative](https://en.wikipedia.org/wiki/Vocative_case). It's the form of the noun you use when speaking to someone. Some languages have a special form of the noun for this.

**Pull request merged**